<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>articleComponentUi</class>
 <widget class="QWidget" name="articleComponentUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="toolLayout"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Selling Article:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="simpleCombo" name="comboSelling"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="advancedTable" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simpleCombo</class>
   <extends>QComboBox</extends>
   <header location="global">simplecombo.h</header>
  </customwidget>
  <customwidget>
   <class>advancedTable</class>
   <extends>QTableView</extends>
   <header>advancedtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboSelling</sender>
   <signal>activated(int)</signal>
   <receiver>articleComponentUi</receiver>
   <slot>sellingChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sellingChanged()</slot>
 </slots>
</ui>
